core/domain/email_manager.py

```diff
@@ -255,3 +285,70 @@ def send_moderator_action_email(
     _send_email(
         recipient_id, sender_id, intent, email_subject, full_email_content,
         bcc_admin=True)
+
+
+def send_role_notification_email(
+        inviter_id, recipient_id, recipient_role, exploration_id,
+        exploration_title):
+    """Sends a email when a new user is given activity rights (Manager, Editor,
+    Viewer) to an exploration by creator of exploration.
+
+    Email will only be sent if recipient wants to receive these emails (i.e.
+    'can_receive_editor_role_email' is set True in recipent's preferences).
+    """
+    # Editor role email body and email subject templates.
+    # pylint: disable=invalid-name
+    editor_role_email_subject_template = (
+        '%s invited you to collaborate on Oppia.org')
+
+    editor_role_email_body_template = (
+        '%s has granted you %s to their learning exploration, '
+        '<a href="http://www.oppia.org/%s">%s</a>'
+        ', on Oppia.org.<br>'
+        'This allows you:<br>'
+        '<ul>%s</ul><br>'
+        'You can find exploration'
```

Comment:
You can find the exploration

(also, missing space at end -- this should also become clear when you write the golden version)